src/modelfree/ppo_and_score.py

```diff
@@ -16,6 +16,18 @@ def default_config():
     }
 
 
+@ppo_and_score_ex.named_config
+def ant_score_config():
+    config = {  # noqa: F841
+        "ppo": {"victim": "/home/neel/multiagent-competition/agent-zoo/" + \
```

Comment:
Don't hardcode paths. This also seems like an outdated way of specifying zoo param locations.